library/src/main/java/com/novoda/downloadmanager/MigrationExtractor.java

```diff
@@ -22,17 +22,19 @@
     private static final int TITLE_COLUMN = 1;
     private static final int MODIFIED_TIMESTAMP_COLUMN = 2;
 
-    private static final String DOWNLOADS_QUERY = "SELECT uri, _data, notificationextras FROM Downloads WHERE batch_id = ?";
+    private static final String DOWNLOADS_QUERY = "SELECT uri, hint, notificationextras FROM Downloads WHERE batch_id = ?";
```

Comment:
`hint` is never empty from v1, it's populated at the time the request is made. `_data` by comparison is only populated when an individual file has completed download.